scripts/func_defs.py

```diff
@@ -112,9 +112,9 @@ def methSegAnnot(files, sampleID):
 def list_final_reports(files, sampleID):
     PATH = DIR_final
     if len(files) == 1:
-        return  PATH+sampleID+"_se_bt2.deduped.sorted_"+ASSEMBLY+"_final.nb.html" #---- single end
+        return  PATH+sampleID+"_se_bt2.deduped.sorted_"+assembly+"_final.html" #---- single end
```

Comment:
Why rename ASSEMBLY to assembly?  `ASSEMBLY` is a global variable, but `assembly` is not defined AFAICS.